builtin/providers/azurerm/config.go

```diff
@@ -99,6 +100,11 @@ type ArmClient struct {
 	serviceBusSubscriptionsClient servicebus.SubscriptionsClient
 
 	keyVaultClient keyvault.VaultsClient
+
+	sqlDatabasesClient               sql.DatabasesClient
+	sqlElasticPoolsClient            sql.ElasticPoolsClient
+	sqlRecommendedElasticPoolsClient sql.RecommendedElasticPoolsClient
+	sqlServersClient                 sql.ServersClient
```

Comment:
I have removed the unused clients except for `sqlElasticPoolsClient` which is used by the resource. I assume, Terraform has deprecated the Riviera SDK and will prefer the Azure RM Go SDK, right?